src/scene-entry-manager.js

```diff
@@ -142,8 +136,8 @@ export default class SceneEntryManager {
 
   exitScene = () => {
     this.scene.exitVR();
-    if (NAF.connection.adapter && NAF.connection.adapter.localMediaStream) {
-      NAF.connection.adapter.localMediaStream.getTracks().forEach(t => t.stop());
+    if (APP.dialog && NAF.connection.adapter.localMediaStream) {
```

Comment:
should it be     `if (APP.dialog && APP.dialog.localMediaStream) {` ?

Reply:
Yes, thank you. Good catch!